bin/init.sh

```diff
@@ -13,6 +13,7 @@ read -p "Port number for new app: " port
 read -p "Replace \`demo\` package name with: " package
 read -p "Repo product: (It's first part of the git repo name. Often a team name) " product_name
 read -p "Repo component: (It's second part of git repo name. Application name) " component_name
+read -p "Slack Channel for build notification " slack_channel
```

Comment:
this should really be handled in the project config, shouldn't be needed in build config any more
https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml#L6

Reply:
ok